dev_tools/scripts/create_new_algorithm.sh

```diff
@@ -342,38 +366,31 @@ then
-    echo "The following CMakeLists have to be modified manually:"                   | tee -a todo_${name}.txt
-    echo "core/CMakeLists.txt"                                                      | tee -a todo_${name}.txt
-    echo "core/test/${source_type}/CMakeLists.txt"                                          | tee -a todo_${name}.txt
-    echo ""                                                                         | tee -a todo_${name}.txt
-    echo "reference/CMakeLists.txt"                                                 | tee -a todo_${name}.txt
-    echo "reference/test/${source_type}/CMakeLists.txt"                                     | tee -a todo_${name}.txt
-    echo ""                                                                         | tee -a todo_${name}.txt
-    echo "omp/CMakeLists.txt"                                                       | tee -a todo_${name}.txt
-    echo "omp/test/${source_type}/CMakeLists.txt"                                           | tee -a todo_${name}.txt
-    echo ""                                                                         | tee -a todo_${name}.txt
-    echo "cuda/CMakeLists.txt"                                                       | tee -a todo_${name}.txt
-    echo "cuda/test/${source_type}/CMakeLists.txt"                                           | tee -a todo_${name}.txt
-    echo ""                                                                         | tee -a todo_${name}.txt
-    echo ""                                                                         | tee -a todo_${name}.txt
-    echo "The following header file has to modified:"                               | tee -a todo_${name}.txt
-    echo "core/device_hooks/common_kernels.inc.cpp"                                 | tee -a todo_${name}.txt
+    echo ""                                                   | tee -a todo_${name}.txt
+    echo "The following CMakeLists have to be modified manually:"| tee -a todo_${name}.txt
+    echo "core/CMakeLists.txt"                                | tee -a todo_${name}.txt
+    echo "core/test/${source_type}/CMakeLists.txt"            | tee -a todo_${name}.txt
+    echo ""                                                   | tee -a todo_${name}.txt
+    echo "reference/CMakeLists.txt"                           | tee -a todo_${name}.txt
+    echo "reference/test/${source_type}/CMakeLists.txt"       | tee -a todo_${name}.txt
+    echo ""                                                   | tee -a todo_${name}.txt
+    echo "omp/CMakeLists.txt"                                 | tee -a todo_${name}.txt
+    echo "omp/test/${source_type}/CMakeLists.txt"             | tee -a todo_${name}.txt
+    echo ""                                                   | tee -a todo_${name}.txt
+    echo "cuda/CMakeLists.txt"                                | tee -a todo_${name}.txt
+    echo "cuda/test/${source_type}/CMakeLists.txt"            | tee -a todo_${name}.txt
```

Comment:
What about the HIP `CMakeLists.txt`? Should the user also modify it, or is it actually fine?

Reply:
Good point, this is taken care of I simply did not update this.